gum/backend-posix/gumexceptor-posix.c

```diff
@@ -17,13 +17,16 @@
 # include "backend-qnx/gumqnx.h"
 #endif
 
+#include <capstone.h>
 #include <signal.h>
 #include <stdlib.h>
 #ifdef HAVE_QNX
 # include <sys/debug.h>
 # include <unix.h>
 #endif
 
+#define X86OP(n) insn->detail->x86.operands[n]
```

Comment:
Defines should be namespaced, i.e. `GUM_X86OP`. Macros like this are typically placed next to where they're used. But in this case this is only used twice, so ISTM we don't really need a macro.